### PR TITLE
Renamed ASNavigationController

### PR DESCRIPTION
**Pull request by zboralski**
To ASDKNavigationController.
This is to avoid conflicting with AuthenticationServices in
Xcode 13

### CODE REVIEW

**Comment by benniebotha** [comment]
@nguyenhuy +1

File: Tests/ASNavigationControllerTests.mm [hunk 1]
```diff
@@ -1,5 +1,5 @@
 //
-//  ASNavigationControllerTests.mm
```
Comment (benniebotha) [inline Tests/ASNavigationControllerTests.mm:2]: Should probably rename the file too.

Comment (rcancro) [reply Tests/ASNavigationControllerTests.mm:2]: Agreed! @zboralski could you add this rename and we can get this merged?

**Comment by rcancro** [comment]
Thanks for this work! I apologize for not responding when you put up the original PR. We wanted to get this landed ASAP since iOS15 is imminent so I copied it and added the test rename [here](https://github.com/TextureGroup/Texture/pull/2020). Again, thank you for the contribution and sorry that we were not on the ball approving it!

**Pull request closed without merging**